lms/static/js/spec/discovery/discovery_factory_spec.js

```diff
@@ -121,7 +121,6 @@ define([
 
         it('loads more', function () {
             var requests = AjaxHelpers.requests(this);
-            jasmine.Clock.useMock();
```

Comment:
This was making the test fail. As far as I can tell, there was no reason to use a fake clock in this test.